app/views/devise/shared/_error_messages.html.haml

```diff
@@ -0,0 +1,9 @@
+- if resource.errors.any?
```

Comment:
please remove this file, no need to have it in the code base

Reply:
Changed accordingly.